resources/zeebe.json

```
@@ -240,6 +240,11 @@
           "name": "propagateAllChildVariables",
           "isAttr": true,
           "type": "Boolean"
+        },
+        {
+          "name": "propagateAllParentVariables",
+          "isAttr": true,
+          "type": "Boolean"
```

```suggestion
          "type": "Boolean",
          "default": true
        }
```

Defaults won't be serialized, and if you decide to go this route, please add respective read tests, too.